### DISCUSSION TITLE
想问下时间线有计划做程序图标显示吗

### DISCUSSION THREAD

**Discussion by q943613777** [shion-app/shion#20] [Ideas]
**就个人之感受而言，在时间线上展现程序图标与程序名，将会更具直观性与明晰性，在体验方面亦会更为优良。**
![image](https://github.com/shion-app/shion/assets/53282874/aa6077d3-1936-47cd-8229-f8c861facfd6)

**Comment by hanaTsuk1** [comment]
后续加上
